### PR TITLE
Updated PR template

### PR DESCRIPTION
**Pull request by AMBudnik**
### Context

We need to inform all PR contributors about the need to sign CLA and follow all Contributing guidelines. 
The following change adds two additional checkboxes to the PR template.

### How has this been tested?

This pull request does require adding tests.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #7307 

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by wojciechczerniak** [comment]
@AMBudnik no reviewer? if it's not ready we should convert this PR to draft

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -21,3 +21,5 @@
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project,
 - [ ] My change requires a change to the documentation.
+- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md)
```
Comment (warpech) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: WDYT about merging this into the first point?

```
- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project
```

**Review by warpech — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (AMBudnik) [reply .github/PULL_REQUEST_TEMPLATE.md]: Hey @warpech. I've made the change. Please re-review.

**Review by warpech — APPROVED** [review]
This PR matches my expectation, but I think that @wojciechczerniak also needs to approve as the owner of this code process in this repo

**Review by wojciechczerniak — APPROVED** [review]
👍

**Pull request merged**